src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,7 +1,12 @@
 # Overview 
 
 Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
-It provides APIs for storing and retrieving application settings.
+It provides APIs for Microsoft [Azure's App Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/).
+
+Developers can use this SDK to interact with their [Configuration Stores](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) in the following ways:
+- Perform basic reads, writes, updates, and deletes
+- Get key-value history
```

Comment:
I would say "get configuration setting history", i.e. you can get history of tags, content type, etc. not just key-value.